### PR TITLE
RE-1617 Add rpc-hummingbird dep-update job

### PR DESCRIPTION
**Pull request by mattt416**
Issue: [RE-1617](https://rpc-openstack.atlassian.net/browse/RE-1617)

### CODE REVIEW

**Comment by mattt416** [comment]
This needs to merge _after_ [this](https://github.com/rcbops/rpc-hummingbird/pull/27) merges.

File: rpc_jobs/rpc_hummingbird.yml [hunk 1]
```diff
@@ -89,3 +89,23 @@
...
     jobs:
       - 'Component-Gate-Trigger_{repo_name}'
+
+- project:
+    name: "rpc-hummingbird-dependency-update"
+    repo_name: "rpc-hummingbird"
+    repo_url: "https://github.com/rcbops/rpc-hummingbird"
+    branch:
+      - "master"
+    jira_project_key: "CEPHSTORA"
```
Comment (git-harry) [inline rpc_jobs/rpc_hummingbird.yml]: `"HUM"`?

Comment (mattt416) [reply rpc_jobs/rpc_hummingbird.yml]: Oops, fixed!

**Comment by mattt416** [comment]
recheck_issue_link

**Pull request merged**